views/interface.js

```diff
@@ -0,0 +1,65 @@
+function getPostData() {
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps").then(response => {
+    response.json().then(post => {
+    post.forEach((element, index, array) => {
+        let peeps = element.body
+        document.getElementById("peeps").append(peeps)
+        document.getElementById("peeps").innerHTML += "<br>";
+        })
+   })
+ })
+}
+
+let button = document.getElementById('find-peep')
+button.addEventListener("click", function() {
+    let area = document.getElementById('peep-id')
+    let id = area.value
+    area.value = ""
+    getSinglePeep(id)
+})
+
+function getSinglePeep(id) {
+    
+    fetch("https://chitter-backend-api-v2.herokuapp.com/peeps" + `/${id}`).then(response => {
+    response.json().then(post => {
+    console.log(post)
+     document.getElementById('peeps').innerHTML = `${post}${post.body}`;
```

Comment:
Currently ${post} here prints the JS object onto the page as a string, could try switching to ${post.user.handle} (I think?) or similar